src/main/java/com/github/webicitybrowser/webicity/core/RenderingEngine.java

```diff
@@ -14,13 +19,24 @@ public interface RenderingEngine {
 	Frame createFrame();
 	
 	AssetLoader getAssetLoader();
+	FetchEngine getFetchEngine();
 	
 	ProtocolRegistry getProtocolRegistry();
 	
 	RendererBackendRegistry getBackendRendererRegistry();
 	
 	public static RenderingEngine create(AssetLoader assetLoader) {
-		return new RenderingEngineImp(assetLoader);
+		return new RenderingEngineImp(assetLoader, new FetchEngineImp(new FetchConnectionPool() {

```

Comment:
Make a dedicated class called HTTPFetchEngine, instead of this anonymous class

Reply:
You mean HTTP fetch connection pool?